pkg/services/navtree/navtreeimpl/admin.go

```diff
@@ -135,6 +135,16 @@ func (s *ServiceImpl) getAdminNode(c *contextmodel.ReqContext) (*navtree.NavLink
 		configNodes = append(configNodes, storage)
 	}
 
+	if s.features.IsEnabled(ctx, featuremgmt.FlagOnPremToCloudMigrations) && hasAccess(ac.EvalPermission(ac.ActionSettingsRead, ac.ScopeSettingsAll)) {
```

Comment:
is this the best way to check for server admin privileges? 🤔

Reply:
I think the best way would be to either check the permissions to all impacted components (eg, dashboards, data sources etc) or introduce a new permission for migration. But if you want to keep it simple and just check for server admin rights, use `hasRole('ServerAdmin')` (from [here](https://github.com/grafana/grafana/blob/main/public/app/core/services/context_srv.ts#L120-L126)) instead.